dominant.js

```diff
@@ -0,0 +1,15 @@
+const dominantDirection = word => {
+    let rtl = 0;
+    let ltr = 0;
+
+    const arr = word.split('').map(item => item.codePointAt());
+
+    for (let script of SCRIPTS) {
```

Comment:
Here, you don't have access to the `SCRIPTS` variable. I understand that it could work in the code sandbox on the Eloquent JS page, but we also should handle other cases like running that file locally.
So maybe add some appropriate comment here, or, even better I think, create an additional variable like `LOCAL_SCRIPTS` and add there some mocked data locally. Then, you would be able to use `LOCAL_SCRIPTS` once you will run that code locally, without access to `SCRIPTS`.